Backend/src/main/java/lt/vu/ads/models/OrderInfo.java

```diff
@@ -0,0 +1,18 @@
+package lt.vu.ads.models;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Date;
+
+@Getter@Setter
+public class OrderInfo {
```

Comment:
OrderInfo is not associated in any way with Order, either OrderInfo should have a foreign key to Order e.g. orderId, or Order should have foreign key to OrderInfo e.g. orderInfoId. OrderStatus is just enum, it doesn't have id and we can't associate Order and OrderInfo through it